### PR TITLE
docs(builder): add root node tip for mountId

### PR DESCRIPTION
**Pull request by chenjiahan**
## Description

Add root node tip for mountId config.

## Types of changes

- [x] Docs change / Dependency upgrade
- [ ] Bug fix
- [ ] New feature / Improvement
- [ ] Refactoring
- [ ] Breaking change

## Checklist

- [ ] I have added changeset via `pnpm run change`.
- [ ] I have updated the documentation.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: packages/document/builder-doc/docs/en/config/html/mountId.md [hunk 1]
```diff
@@ -29,6 +29,19 @@ After compilation:
...
+### Notes
+
+#### Update Relevant Code
+
+After modifying `mountId`, if there is logic in your code to obtain the `root` root node, please update the corresponding value:
+
+```diff
+- const domNode = document. getElementById('root');
++ const domNode = document. getElementById('app');
+
```
Comment (fireairforce) [inline packages/document/builder-doc/docs/en/config/html/mountId.md:41]:
```suggestion
const domNode = document. getElementById('app');
const domNode = document.getElementById('app');
```
多了个空格

File: packages/document/builder-doc/docs/en/config/html/mountId.md [hunk 2]
```diff
@@ -29,6 +29,20 @@ After compilation:
...
-:::
+### Notes
+
+#### Update Relevant Code
+
+After modifying `mountId`, if there is logic in your code to obtain the `root` root node, please update the corresponding value:
+
+```diff
+- const domNode = document. getElementById('root');
++ const domNode = document. getElementById('app');
```
Comment (chenjiahan) [inline packages/document/builder-doc/docs/en/config/html/mountId.md:40]:
```suggestion
+ const domNode = document.getElementById('app');
```

File: packages/document/builder-doc/docs/en/config/html/mountId.md [hunk 3]
```diff
@@ -29,6 +29,20 @@ After compilation:
...
-If you customized the HTML template, please make sure that the template contains `<div id="<%= mountId %>"></div>`, otherwise this config will not take effect.
-:::
+### Notes
+
+#### Update Relevant Code
+
+After modifying `mountId`, if there is logic in your code to obtain the `root` root node, please update the corresponding value:
+
+```diff
+- const domNode = document. getElementById('root');
```
Comment (chenjiahan) [inline packages/document/builder-doc/docs/en/config/html/mountId.md:39]:
```suggestion
- const domNode = document.getElementById('root');
```

**Review by fireairforce — APPROVED** [review]
LGTM

**Pull request merged**